test/integration/manifest_int_test.go

```diff
@@ -41,6 +42,66 @@ func (suite *ManifestIntegrationTestSuite) TestManifest() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *ManifestIntegrationTestSuite) TestManifest_JSON() {
+	suite.OnlyRunForTags(tagsuite.Manifest)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
+
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState/cli#9eee7512-b2ab-4600-b78b-ab0cf2e817d8", "."),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
+	)
+	cp.Expect("Checked out project", e2e.RuntimeSourcingTimeoutOpt)
+
+	type version struct {
+		Requested string `json:"requested"`
+		Resolved  string `json:"resolved"`
+	}
+
+	type vulnerabilities struct {
+		Count map[string]int `json:"count"`
+	}
+
+	type requirement struct {
+		Name            string          `json:"name"`
+		Version         version         `json:"version"`
+		Vulnerabilities vulnerabilities `json:"vulnerabilities"`
+	}
+
+	type requirements struct {
+		Requirements []requirement `json:"requirements"`
+	}
```

Comment:
This could end up being painful to maintain as it makes us very opinionated on the exact format of the json output which isn't exactly what we intend to test here.

Suggest instead using https://github.com/ActiveState/cli/blob/DX-2720/test/integration/shared_int_test.go#L26-L26. I don't think you need to assert the exact version numbers or counts, maybe just check that we are producing output with the `requirements` key or something. 

I think this test should assert that we are in fact outputting json, and that the outputted json isn't an error message.

Another option is to simply expose the structs you used to generate the JSON, and unmarshal from that.